pages/learn/welcome/primer.md

```diff
@@ -5,7 +5,7 @@ excerpt: How {{ $names.company.lower }} gets your code to your device fleet, fro
 
 # A {{ $names.company.lower }} primer
 
-The {{ $names.company.lower }} platform encompasses device, server, and client software, all designed to get your code securely deployed to a fleet of devices. The broad strokes are easy to grasp: once your device is set up with our [host OS][os-docs], you can push code to the {{ $names.company.lower }} [build servers][build], where it will be packaged into containers and delivered to your fleet. All your devices and their services can be managed, monitored, and updated from your [{{ $names.cloud.lower }} dashboard][dashboard].
+The {{ $names.company.lower }} platform encompasses device, server, and client software, all designed to get your code securely deployed to a fleet of devices. The broad strokes are easy to grasp: once your device is set up with our [host OS][os-docs], you can push code to the {{ $names.company.lower }} [servers][build], where it will be packaged into containers and delivered to your fleet. All your devices and their services can be managed, monitored, and updated from your [{{ $names.cloud.lower }} dashboard][dashboard].
```

Comment:
Any reasoning as to why we are removing the word "build" though. Throughout the docs, the servers building releases, packaging the containers, and handling the backend process are called build servers.